src/domain/enrolment/utils.ts

```diff
@@ -5,15 +5,10 @@ import {
 } from '../../generated/graphql';
 import { EnrolmentFormFields } from './enrolmentForm/constants';
 
-const getNotificationType = (
-  values: EnrolmentFormFields
-): NotificationType | undefined => {
+const getNotificationType = (values: EnrolmentFormFields): NotificationType => {
```

Comment:
```
ternary might read nicer in this situation :)

return values.hasEmailNotification && values.hasSmsNotification
    ? NotificationType.EmailSms
    : NotificationType.Email;
```

Reply:
true that. Because of some reason I wanted to keep it easily recoverable / extendable :)

Reply:
Well it wasn't wrong :)